internal/common/update/patch_payload_test.go

```diff
@@ -187,14 +188,33 @@ func TestPatchReplicationSpecs(t *testing.T) {
 				plan:          &planNoChanges,
 				patchExpected: nil,
 			},
+			"Empty array should return no changes": {
+				state: &admin.ClusterDescription20240805{
+					Labels: &[]admin.ComponentLabel{},
+				},
+				plan: &admin.ClusterDescription20240805{
+					Labels: &[]admin.ComponentLabel{},
+				},
+				patchExpected: nil,
+			},
+			"diskSizeGb ignored in state": {
+				state:         clusterDescriptionDiskSizeNodeCount(50.0, 3, conversion.Pointer(50.0), 0),
+				plan:          clusterDescriptionDiskSizeNodeCount(55.0, 3, nil, 0),
+				patchExpected: clusterDescriptionDiskSizeNodeCount(55.0, 3, nil, 0),
+				options: []update.PatchOptions{
+					{
+						IgnoreInState: []string{"diskSizeGB"},
+					},
+				},
+			},
 		}
 	)
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
 			if name == "Removed list entry should be included" {
 				t.Log("This test case is expected to fail due to the current implementation")
```

Comment:
is this log still accurate?  is that test failing?

Reply:
Good observation. Removed